service/src/main/java/eu/europa/ec/fisheries/uvms/reporting/service/bean/impl/ReportingConfigProducerBean.java

```diff
@@ -40,7 +40,7 @@ public String sendConfigMessage(String textMsg) {
         try {
             return sendModuleMessage(textMsg, reportingINQueue);
         } catch (MessageException e) {
-            log.error("[ERROR] Error while trying to send message to Config! Check ReportingConfigProducerBeanImpl..");
+            log.error("[ERROR] Error while trying to send message to Config! Check ReportingConfigProducerBeanImpl..",e);
```

Comment:
Remove the `"[ERROR] "` part of the message, it is redundant and confusing in the logs.